ompi/mca/coll/accelerator/coll_accelerator_component.c

```diff
@@ -88,5 +93,32 @@ static int accelerator_register(void)
                                            MCA_BASE_VAR_SCOPE_READONLY,
                                            &mca_coll_accelerator_component.disable_accelerator_coll);
 
+    mca_coll_accelerator_bcast_thresh = 256;
+    (void) mca_base_component_var_register(&mca_coll_accelerator_component.super.collm_version,
+                                           "bcast_thresh",
+                                           "max. msg length for which to copy accelerator buffer to CPU for bcast operation",
+                                           MCA_BASE_VAR_TYPE_INT, NULL, 0, 0,
+                                           OPAL_INFO_LVL_9,
+                                           MCA_BASE_VAR_SCOPE_READONLY,
+                                           &mca_coll_accelerator_bcast_thresh);
+
+    mca_coll_accelerator_allgather_thresh = 65536;
+    (void) mca_base_component_var_register(&mca_coll_accelerator_component.super.collm_version,
+                                           "allgather_thresh",
+                                           "max. overall msg length for which to copy accelerator buffer to CPU for allgather operation",
```

Comment:
I'm not sure that `overall msg length` communicates well that the threshold is compared to `send_size*comm_size`. See my comment above for my concerns about using the process count.